src/grt/src/fastroute/include/FastRoute.h

```diff
@@ -545,6 +556,15 @@ class FastRouteCore
 
   FastRouteRenderer* fastrouteRender_;
   std::unique_ptr<DebugSetting> debug_;
+
+  std::unordered_map<std::tuple<int, int, int>,
```

Comment:
Prefer a struct to a tuple as it is clear what the three ints represent from this declaration